ubidots/ubidots_in.html

```diff
@@ -155,7 +155,7 @@
       <ul>
           <li>
             <p>
-              <b> Account Type: </b>Defaults to "Ubidots" (Paid Account). If you have an education account choose "Ubidots for Education" (STEM account).
+              <b> Account Type: </b>Defaults to “Ubidots”, which is valid for both industrial and stem account type. If you have an educational account, choose “Ubidots for Education”
```

Comment:
Not sure if we should better talk about `Industrial` and `STEM` here (notice the capitalization and uppercase) to keep the branding.